packages/autorest.python/autorest/codegen/models/base_builder.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         self.api_versions: List[str] = yaml_data["apiVersions"]
         self.added_on: Optional[str] = yaml_data.get("addedOn")
         self.external_docs: Optional[Dict[str, Any]] = yaml_data.get("externalDocs")
+        self.has_native_overload: bool = yaml_data.get("hasNativeOverload", False)
```

Comment:
is native overload an overload directly from cadl?

Reply:
@iscai-msft Yeah, that's why we call it "native"